WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -145,16 +145,16 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
 
-    self.tableView = [[UITableView alloc] initWithFrame:CGRectZero style:UITableViewStylePlain];
-    self.tableView.translatesAutoresizingMaskIntoConstraints = NO;
-    [self.view addSubview:self.tableView];
-    [self.view pinSubviewToAllEdges:self.tableView];
+    [self configureStackView];
 
-    [WPStyleGuide configureColorsForView:self.view andTableView:self.tableView];
-    self.editButtonItem.accessibilityIdentifier = NSLocalizedString(@"Edit", @"");
+    [self configureSearchBar];
+    [self.stackView insertArrangedSubview:self.searchBar atIndex:0];
```

Comment:
You could just do `addArrangedSubview:` here and for the table view, and it'll insert them in order.